### ISSUE TITLE
After running install_beamsearch_decoders.sh, testing script starts endless loop.

### ISSUE THREAD

**Issue by Varuzhan97** [NVIDIA/NeMo#7595]
After running install_beamsearch_decoders.sh, testing script starts endless loop. I am using Ubuntu 22.04 with Python 3.10. 
The loop is somewhere in this code:

       `with self.autocast():
            with torch.no_grad():
                if isinstance(self.asr_model, EncDecHybridRNNTCTCModel):
                    self.asr_model.cur_decoder = 'ctc'
                all_logits = self.asr_model.transcribe([audio_file], logprobs=True, verbose = 0)

        preds = np.argmax(all_logits[0], axis=1)
        preds_tensor = torch.tensor(preds, device=self._device).unsqueeze(0)
        if isinstance(self.asr_model, EncDecHybridRNNTCTCModel):
            pred_text = self.asr_model.ctc_decoding.ctc_decoder_predictions_tensor(preds_tensor)[0][0]
        else:
            pred_text = self.asr_model._wer.decoding.ctc_decoder_predictions_tensor(preds_tensor)[0][0]

        #self.asr_model = self.asr_model.to('cpu')

        decoding = self.toggle_decoder("beam")

        probs_lens = torch.tensor([all_logits[0].shape[0]])

        with torch.no_grad():
            packed_batch = torch.cat([torch.tensor(all_logits[0], device=self._device, dtype=torch.float32).unsqueeze(0)], dim=1)
       hypotheses = decoding.ctc_decoder_predictions_tensor(packed_batch, decoder_lengths=probs_lens, return_hypotheses=True)`

I also tried to setup environment on Python 3.8, 3.9 and 3.11, either gives "install_beamsearch_decoders.sh not installed " error, or starts endless loop.
